bin/tests/check_dl_core_callbacks.sh

```diff
@@ -789,4 +789,36 @@ assert metrics['last']['cmc01'] > 0.1  # slightly better then random
 assert metrics['last']['cmc05'] > 0.5
 """
 
+################################  pipeline 22  ################################
+# test autoresume
+
+LOG_MSG='pipeline 22'
+echo ${LOG_MSG}
+
+EXPDIR=./tests/_tests_dl_callbacks
+LOGDIR=./tests/logs/_tests_dl_callbacks
+CHECKPOINTS=${LOGDIR}/checkpoints
+LOGFILE=${CHECKPOINTS}/_metrics.json
+EXP_OUTPUT=./tests/output.txt
+
+INTERRUPT_EPOCH=4 \
```

Comment:
could we use `--check` arg instead?

Reply:
hmm, config already have CheckRunCallback, i think it's a same thing